HouseRules_Essentials/README.md

```diff
@@ -725,28 +726,28 @@ The [Settings Reference](../docs/SettingsReference.md) contains lists of all dif
     "Rule": "StartCardsModified",
     "Config": {
       "HeroGuardian": [
-        { "Card": "HealingPotion", "IsReplenishable": false },
-        { "Card": "ReplenishArmor", "IsReplenishable": true },
-        { "Card": "WhirlwindAttack", "IsReplenishable": true },
-        { "Card": "PiercingThrow", "IsReplenishable": false },
-        { "Card": "CoinFlip", "IsReplenishable": false },
-        { "Card": "TheBehemoth", "IsReplenishable": false },
-        { "Card": "SwordOfAvalon", "IsReplenishable": false },
+        { "Card": "HealingPotion", "IsReplenishable": 0 },
```

Comment:
Does it make sense to switch from `IsReplenishable` to something less boolean-sounding in a later patch?  E.g., `Cooldown`, `ReplenishableRounds`, etc.?